crates/hyperswitch_domain_models/src/payments/payment_attempt.rs

```diff
@@ -1405,6 +1405,18 @@ impl PaymentAttemptUpdate {
     }
 }
 
+#[cfg(feature = "v2")]
+#[derive(Debug, Clone, Serialize)]
+pub struct ConfirmIntentResponseUpdate {
```

Comment:
Had to move this as a separate struct because of the `clippy::large_enum_variant` lint being thrown.